/var/lib/slurm/slurmd/job10213/slurm_script: line 16: activate: No such file or directory

CondaError: Run 'conda init' before 'conda deactivate'


CondaError: Run 'conda init' before 'conda activate'

[2024-06-02 17:06:45,132][    INFO] {'batch_size': 12,
 'conf_thresh': 0.95,
 'config': 'configs/ukbb/train/exp4/seed53/config.yaml',
 'crop_size': 256,
 'ctrl': 'age sex ethnicity',
 'data_desc': 'UKBB Train Data - 182 Patients - 18 Labeled, Rest Unlabeled',
 'data_root': '/vol/aimspace/projects/ukbb/data/cardiac/cardiac_segmentations/subjects',
 'dataset': 'ukbb',
 'epochs': 1,
 'local_rank': 0,
 'lr': 0.01,
 'mode': 'train',
 'model_desc': 'Unimatch Unet 18 Training on UKBB',
 'nclass': 4,
 'ngpus': 1,
 'num_workers': 1,
 'seed': 53,
 'split': 18,
 'visualize': False}

[2024-06-02 17:06:45,182][    INFO] Total params: 1.8M

[2024-06-02 17:06:47,370][    INFO] ===========> Epoch: 0, LR: 0.01000, Previous best: 0.00
[2024-06-02 17:06:50,514][    INFO] torch.Size([12, 4, 256, 256])
[2024-06-02 17:06:50,514][    INFO] torch.Size([12, 4, 256, 256])
Traceback (most recent call last):
  File "/vol/aimspace/users/khattab/UniMatch/more-scenarios/medical/unimatch_ukbb.py", line 304, in <module>
    main()
  File "/vol/aimspace/users/khattab/UniMatch/more-scenarios/medical/unimatch_ukbb.py", line 178, in main
    p_ratio = (1e-6 + p_data().cuda()) / (1e-6 + p_model().cuda())
  File "/vol/aimspace/users/khattab/UniMatch/more-scenarios/medical/util/dist_align.py", line 22, in __call__
    return v / torch.sum(v, keepdim=True)
TypeError: sum() received an invalid combination of arguments - got (Tensor, keepdim=bool), but expected one of:
 * (Tensor input, *, torch.dtype dtype)
 * (Tensor input, tuple of ints dim, bool keepdim, *, torch.dtype dtype, Tensor out)
 * (Tensor input, tuple of names dim, bool keepdim, *, torch.dtype dtype, Tensor out)

